### DISCUSSION TITLE
Checkbox - required is not working as select anyone

### DISCUSSION THREAD

**Discussion by mary-josephine-1** [buefy/buefy#3943] [Q&A]
### Overview of the problem

**OS/Browser**:Ubuntu
```
    "nuxt": "^2.15.8",
    "nuxt-buefy": "^0.4.26",
    "vue": "^2.7.10",
```

### Description
Array based checkbox is not as expected. It forces to select all items instead of any in the list to continue.

### Steps to reproduce

```
 <form @submit.prevent="submitForm()">
    <section>
      <b-field label="testing"> </b-field>
      <template v-for="(option, o) in checkboxItems">
        <b-checkbox :key="o" v-model="selected" :native-value="option" required>
          {{ option }}
        </b-checkbox>
      </template>
      <br>
      <br>
      selected : {{selected}}
    </section>

    <button type="submit"> confirm </button>
  </form>
```
DATA
```
 data() {
    return {
      checkboxItems: ["A", "B", "C", "D"],
      selected: []
    };
  }
```

### Expected behavior

Select only the required ones.

### Actual behavior

All items in the list must be selected.

**Comment by kikuomax** [comment]
@mary-josephine-1 According to your code snippet all the checkboxes are `required`, so I think the actual behavior is correct; i.e., all the checkboxes must be checked.

If you intend to require at least one of the listed checkboxes to be checked, [this StackOverflow entry](https://stackoverflow.com/questions/22238368/how-can-i-require-at-least-one-checkbox-be-checked-before-a-form-can-be-submitte) might help.
